core/src/main/java/it/pagopa/selfcare/mscore/core/strategy/factory/CreateInstitutionStrategyFactory.java

```diff
@@ -133,7 +133,7 @@ private Function<CreateInstitutionStrategyInput, Institution> mappingToInstituti
             newInstitution.setSubunitCode(strategyInput.getSubunitCode());
             newInstitution.setSubunitType(InstitutionPaSubunitType.AOO.name());
             newInstitution.setParentDescription(institutionProxyInfo.getDescription());
-
+            newInstitution.setRootParentId(institutionProxyInfo.getId());
```

Comment:
maybe the rootParentId shouldn't be taken from institutionProxyInfo but from institution after saving...